integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/noarkexchange/receive/InternalQueue.java

```diff
@@ -186,6 +188,23 @@ public void dlqListener(byte[] message, Session session) {
         conversationService.registerStatus(conversationId, ms);
     }
 
+    private void writeMessageToDisk(EDUCore request) {
+        PutMessageRequestType putMessage = EDUCoreFactory.createPutMessageFromCore(request);
+        try {
+            Marshaller marshaller = jaxbContextdomain.createMarshaller();
+            ByteArrayOutputStream bos = new ByteArrayOutputStream();
+            marshaller.marshal(new JAXBElement<>(new QName("uri", "local"), PutMessageRequestType.class, putMessage), bos);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "local" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1192)